src/Products/Common/Resources/Resources.cs

```diff
@@ -72,12 +71,12 @@ public ExceptionEntity GenerateException(System.Exception ex, String password)
             // Initiate exception
             ExceptionEntity exceptionEntity = new ExceptionEntity();
             // Check if exception message contains password and password is empty
-            if (ex.Message.Contains("password") && String.IsNullOrEmpty(password))
+            if (ex.Message.ToLower().Contains("password") && String.IsNullOrEmpty(password))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-MVC/pullRequest?prid=5133239)